### PR TITLE
Add EZ PHP 7.1 image

### PR DESCRIPTION
**Pull request by kierenevans**
EZ Platform uses Symfony framework (not just Symfony components), so will benefit from using the Symfony images as the base images.

### CODE REVIEW

File: ez/6.x/Dockerfile-php7.1 [hunk 1]
```diff
@@ -0,0 +1,6 @@
+FROM quay.io/continuouspipe/symfony-php7.1-apache:latest
```
Comment (andytson-inviqa) [inline ez/6.x/Dockerfile-php7.1]: how much of Symfony does it use? If not supporting the symfony bundle system I think we should do as I did with Spryker. All the symfony_functions.sh assume there's the symfony console tool

File: ez/6.x/Dockerfile.tmpl [hunk 2]
```diff
@@ -0,0 +1,6 @@
+FROM quay.io/continuouspipe/symfony-php{{PHP_VERSION}}-apache:latest
```
Comment (andytson-inviqa) [inline ez/6.x/Dockerfile.tmpl]: how much of Symfony does it use? If not supporting the symfony bundle system I think we should do as I did with Spryker. All the symfony_functions.sh assume there's the symfony console tool

Comment (andytson-inviqa) [inline ez/6.x/Dockerfile.tmpl]: looks like it does use it properly, so can ignore this

File: continuous-pipe.yml [hunk 3]
```diff
@@ -110,6 +116,9 @@ tasks:
         ez:
```
Comment (andytson-inviqa) [inline continuous-pipe.yml]: would be good to rename this one for consistency

File: ez/6.x/Dockerfile.tmpl (hunk 2 above)
Comment (kierenevans) [inline ez/6.x/Dockerfile.tmpl]: Yep, very much a framework built from the full symfony stack.

File: continuous-pipe.yml (hunk 3 above)
Comment (kierenevans) [inline continuous-pipe.yml]: Updated

**Pull request merged**